tutorials/metrics.md

```diff
@@ -39,22 +39,34 @@ metrics_key = "/path/to/key.pem"
+| `crio_containers_oom_total`                      |                                                                                                                                         | Counter   | Total number of containers killed because they ran out of memory (OOM)                                            |
+| `crio_containers_oom`                            | `name`                                                                                                                                  | Counter   | (DEPRECATED) Containers killed because they ran out of memory (OOM) by their name                                 |
+| `crio_processes_defunct`                         |                                                                                                                                         | Gauge     | Total number of defunct processes in the node                                                                     |
+| `crio_operations_total`                          | every CRI-O RPC\*                                                                                                                       | Counter   | Cumulative number of CRI-O operations by operation type.                                                          |
+| `crio_operations_latency_seconds_total`          | every CRI-O RPC\*,<br><br>`network_setup_pod` (CNI pod network setup time),<br><br>`network_setup_overall` (Overall network setup time) | Summary   | Latency in seconds of CRI-O operations. Split-up by operation type.                                               |
+| `crio_operations_latency_seconds`                | every CRI-O RPC\*                                                                                                                       | Gauge     | Latency in microseconds of individual CRI calls for CRI-O operations. Broken down by operation type.              |
+| `crio_operations_errors_total`                   | every CRI-O RPC\*                                                                                                                       | Counter   | Cumulative number of CRI-O operation errors by operation type.                                                    |
+| `crio_image_pulls_by_digest_bytes_total`         | `name`, `digest`, `mediatype`, `size`                                                                                                   | Counter   | Bytes transferred by CRI-O image pulls by digest.                                                                 |
+| `crio_image_pulls_by_name_bytes_total`           | `name`, `size`                                                                                                                          | Counter   | Bytes transferred by CRI-O image pulls by name.                                                                   |
```

Comment:
Isn't this identical to `sum by (name, size) (crio_image_pulls_by_digest_bytes_total)`? In that case it's redundant.

Reply:
Yes, this is kinda redundant. @saschagrunert can we go ahead and deprecate `crio_image_pulls_by_name` altogether instead of introducing `crio_image_pulls_by_name_bytes_total` as it's canonical.

Reply:
> Isn't this identical to sum by (name, size) (crio_image_pulls_by_digest_bytes_total)? In that case it's redundant.

good point, though if we end up reconsidering listing each by digest as we're discussing in https://github.com/cri-o/cri-o/pull/5487#discussion_r763360192 then we should keep.

Reply:
also won't all the _bytes fields have a large cardinality? maybe we should sort into buckets...

Reply:
> also won't all the _bytes fields have a large cardinality? maybe we should sort into buckets...

Yes, similar to previous comment. 
Also, should be remove bytes label as well?

Reply:
Introduced metric names as 
`crio_image_pulls_bytes_total` and `crio_image_pulls_skipped_bytes_total` that follow naming best practices. Dropped high cardinality labels like `name`, `digest`. However, `size` is being used but with sorted buckets so as to keep cardinality in check.